src/java.base/share/classes/java/util/List.java

```diff
@@ -781,6 +780,76 @@ default Spliterator<E> spliterator() {
         }
     }
 
+    // ========== SequencedCollection ==========
+
+    /**
+     * {@inheritDoc}
+     * @since 21
+     */
+    default void addFirst(E e) { this.add(0, e); }
+
+    /**
+     * {@inheritDoc}
+     * @since 21
+     */
+    default void addLast(E e) { this.add(e); }
+
+    /**
+     * {@inheritDoc}
+     * @since 21
+     */
+    default E getFirst() {
+        if (this.isEmpty())
```

Comment:
weirdly, sometimes you use braces around the ''if and sometimes you don't ?

Reply:
Yes, some of the code here is badly formatted. Laziness on my part.